### PR TITLE
fix(mobile): component alignment issue on left side of login page (#899)

### LINKED ISSUE #899: Component alignment issue on Left Side of the login page
### Reproduction / Replay Link (Optional)

_No response_

### Description

**Steps to reproduce:**

1. Go to the login page at https://www.openstatus.dev/app/login.
2. Look at the content on the left side.

**Expected behavior:**

The content on the left side should be center-aligned.

**Actual behavior:**

The content on the left side is not center-aligned, as shown in the attached screenshot.

**Screenshot:**

Alignment Issue

### Environment

_No response_

### PR DESCRIPTION
**Pull request by tsahil01**
## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation

### Fixes Alignment Issue on Left Side of Login Page (#899)

Fixed alignment issue on the left side of the login page at https://www.openstatus.dev/app/login. Content is now center-aligned and text sizes adjusted for improved visual consistency.

## Before this PR
Before
#### sm devices -
Before-mobile-view

## After this PR
After
#### sm devices -
image

### Related Issue (optional)

Fixes #899

### CODE REVIEW

**Comment by mxkaske** [comment]
Hey @tsahil01! I agree with you, centering the text on mobile feels much better! On larger device, I see it less as a problem but that’s just a personal preference. What do you think?

Thanks for improving OpenStatus.

**Comment by tsahil01** [comment]
Hi @mxkaske! Thanks for the feedback! I believe centring the text enhances the user experience across all device sizes, making it more consistent and visually appealing.

What do you think about merging the PR as it is? 
I'm also open to making further improvements based on your suggestions.

File: apps/web/src/app/app/(auth)/login/layout.tsx [hunk 1]
```diff
@@ -24,12 +24,12 @@ export default async function AuthLayout({
...
         </Link>
-        <div className="flex w-full max-w-lg flex-1 flex-col justify-center gap-8 text-left">
+        <div className="flex w-full max-w-lg flex-1 flex-col justify-center gap-8 text-center mx-auto">
           <div className="mx-auto grid gap-3">
-            <h1 className="font-cal text-2xl text-foreground">
+            <h1 className="font-cal text-3xl text-foreground">
               Open Source Monitoring Service
             </h1>
-            <p className="text-muted-foreground">
+            <p className="text-muted-foreground text-sm mx-9">
```
Comment (mxkaske) [inline apps/web/src/app/app/(auth)/login/layout.tsx]:
```suggestion
            <p className="text-muted-foreground text-sm">
```

File: apps/web/src/app/app/(auth)/login/layout.tsx [hunk 2]
```diff
@@ -24,12 +24,12 @@ export default async function AuthLayout({
             className="rounded-full border border-border"
           />
         </Link>
-        <div className="flex w-full max-w-lg flex-1 flex-col justify-center gap-8 text-left">
+        <div className="flex w-full max-w-lg flex-1 flex-col justify-center gap-8 text-center mx-auto">
```
Comment (mxkaske) [inline apps/web/src/app/app/(auth)/login/layout.tsx]:
```suggestion
        <div className="flex w-full max-w-lg flex-1 flex-col justify-center gap-8 text-center mx-auto md:text-left md:mx-0">
```

**Review by mxkaske — COMMENTED** [review]
Let's first start with mobile/tablet version if you are fine with it.

**Comment by tsahil01** [comment]
Hi @mxkaske , I'll implement the mobile/tablet changes and push the updates soon. 
Thanks!

**Comment by tsahil01** [comment]
I've implemented the mobile/tablet version changes as suggested and pushed the updates.
### Mobile/Tablet - 
image

### Larger screen -
image

**Comment by mxkaske** [comment]
Appreciate it! Merged!

**Pull request merged**